.github/workflows/lint.yml

```diff
@@ -0,0 +1,15 @@
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "3.2.1"
```

Comment:
This will get picked up automatically from `.tool-versions` FWIW

Reply:
TIL. fixing now!